legend-sdlc-generation-service/pom.xml

```diff
@@ -183,6 +188,11 @@
             <artifactId>jetty-servlet</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Actually, it looks like this needs to have compile scope after all, since legend-pure-m3-core has a compile scope dependency on it.

Reply:
I've fixed this on the latest commit.